react/src/components/BatchSessionScheduledTimeSetting.tsx

```diff
@@ -57,8 +61,8 @@ const BatchSessionScheduledTimeSetting: React.FC<Props> = ({
             onChange={(value) => {
               dispatchAndSetScheduleTime(value);
             }}
-            onBlur={() => {
-              dispatchAndSetScheduleTime(scheduleTime);
+            onCalendarChange={() => {
+              datePickerRef.current?.focus();
```

Comment:
Clicking on the button to flip the month closes the date picker. Using onPanelChange seems to solve it 
```
  onPanelChange={()=>{
              datePickerRef.current?.focus(); 
            }}
```